app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="app_name">Super Converge Lvlup</string>
```

Comment:
@omokolataiwo 
Please try rename the app name to something more reasonable and befitting to the kind of application you are building.
Thanks.